udata/tests/frontend/test_reuse_frontend.py

```diff
@@ -48,9 +48,26 @@ def test_render_list_empty(self):
 
     def test_render_display(self):
         '''It should render the reuse page'''
-        reuse = ReuseFactory()
-        response = self.get(url_for('reuses.show', reuse=reuse))
+        author = UserFactory()
+        reuse = ReuseFactory(url='http://www.datagouv.fr/reuse',
```

Comment:
OK, I removed all the attributes which were populated by the factory (except the markdown description).